### ISSUE TITLE
Error after updating from 0.6.2 to 0.6.3: "Unsupported role MessagesPlaceholder"

### ISSUE THREAD

**Issue by sammrai** [Chainlit/chainlit#316]
**Description:**

After updating from version 0.6.2 to 0.6.3, I encountered an error related to an "Unsupported role MessagesPlaceholder". The traceback provided below gives more details about the issue.

**Error Traceback:**

```
chainlit-demo_1  | > Entering new AgentExecutor chain...
chainlit-demo_1  | 2023-08-25 04:47:54 - Error in LangchainCallbackHandler.on_chain_start callback: Unsupported role MessagesPlaceholder
chainlit-demo_1  | 2023-08-25 04:47:54 - Unsupported role MessagesPlaceholder
chainlit-demo_1  | Traceback (most recent call last):
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/chainlit/utils.py", line 36, in wrapper
chainlit-demo_1  |     return await user_function(**params_values)
chainlit-demo_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/demo_app/main.py", line 46, in main
chainlit-demo_1  |     res = llm_chain.run(message, callbacks=[cl.LangchainCallbackHandler()])
chainlit-demo_1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/chains/base.py", line 475, in run
chainlit-demo_1  |     return self(args[0], callbacks=callbacks, tags=tags, metadata=metadata)[
chainlit-demo_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/chains/base.py", line 282, in __call__
chainlit-demo_1  |     raise e
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/chains/base.py", line 276, in __call__
chainlit-demo_1  |     self._call(inputs, run_manager=run_manager)
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/agents/agent.py", line 1036, in _call
chainlit-demo_1  |     next_step_output = self._take_next_step(
chainlit-demo_1  |                        ^^^^^^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/agents/agent.py", line 833, in _take_next_step
chainlit-demo_1  |     output = self.agent.plan(
chainlit-demo_1  |              ^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/agents/agent.py", line 456, in plan
chainlit-demo_1  |     full_output = self.llm_chain.predict(callbacks=callbacks, **full_inputs)
chainlit-demo_1  |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/chains/llm.py", line 255, in predict
chainlit-demo_1  |     return self(kwargs, callbacks=callbacks)[self.output_key]
chainlit-demo_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/chains/base.py", line 270, in __call__
chainlit-demo_1  |     run_manager = callback_manager.on_chain_start(
chainlit-demo_1  |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/callbacks/manager.py", line 1166, in on_chain_start
chainlit-demo_1  |     _handle_event(
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/callbacks/manager.py", line 328, in _handle_event
chainlit-demo_1  |     raise e
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/langchain/callbacks/manager.py", line 304, in _handle_event
chainlit-demo_1  |     getattr(handler, event_name)(*args, **kwargs)
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/chainlit/langchain/callbacks.py", line 378, in on_chain_start
chainlit-demo_1  |     prompt = build_prompt(serialized, inputs)
chainlit-demo_1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/chainlit/langchain/callbacks.py", line 45, in build_prompt
chainlit-demo_1  |     role=convert_role(m["id"][-1]),
chainlit-demo_1  |          ^^^^^^^^^^^^^^^^^^^^^^^^^
chainlit-demo_1  |   File "/app/.venv/lib/python3.11/site-packages/chainlit/langchain/callbacks.py", line 74, in convert_role
chainlit-demo_1  |     raise ValueError(f"Unsupported role {role}")
chainlit-demo_1  | ValueError: Unsupported role MessagesPlaceholder
```

```python:main.py
# main.py
from langchain.chat_models import ChatOpenAI
from langchain.agents import Tool
from langchain.chains.conversation.memory import ConversationBufferMemory
from langchain.agents import initialize_agent
from langchain.agents import AgentType
from langchain.utilities import BingSearchAPIWrapper
import chainlit as cl

search = BingSearchAPIWrapper()
tools = [
    Tool(
        name = "Current Search",
        func=search.run,
        description="useful for when you need to answer questions about current events or the current state of the world"
    ),
]

@cl.on_chat_start
def main():
    memory = ConversationBufferMemory(memory_key="chat_history", return_messages=True)
    llm = ChatOpenAI(temperature=0)

    agent_chain = initialize_agent(
        tools,
        llm,
        agent=AgentType.CHAT_CONVERSATIONAL_REACT_DESCRIPTION,
        verbose=True,
        memory=memory
    )

    cl.user_session.set("llm_chain", agent_chain)


@cl.on_message
async def main(message: str):
    llm_chain = cl.user_session.get("llm_chain")

    res = llm_chain.run(message, callbacks=[cl.LangchainCallbackHandler()])

    await cl.Message(content=res).send()
```

**Steps to Reproduce:**

1. Update from version 0.6.2 to 0.6.3.
2. Run the main.py application.
3. Observe the error in the logs.

**Expected Behavior:**

The application should run without any errors after the update.

**Actual Behavior:**

Received an error related to "Unsupported role MessagesPlaceholder".

**Environment:**
- python = "3.11.4"
- openai = "0.27.9"
- chainlit = "0.6.3"
- langchain = "0.0.272"

**Comment by willydouhard** [comment]
This is a bug, nice catch!

Will be fixed in the next release.

**Comment by willydouhard** [comment]
Should be fixed with the `0.6.401` release!
